DNN Platform/Library/Data/SqlDataProvider.cs

```diff
@@ -442,13 +442,13 @@ public override IDataReader ExecuteSQL(string sql, int timeoutSec)
         {
             return ExecuteSQLInternal(ConnectionString, sql, timeoutSec);
         }
-
```

Comment:
please exclude these non beneficial file changes from your pull request

Reply:
@sleupold I think it's worthy for readability to add one line space between two methods definitions